packages/patterns/objects/o-nav/o-nav.prototype.js

```diff
@@ -0,0 +1,22 @@
+const path = require( 'path' );
+const clone = require( '../../../larva/lib/utils/clonePatternData' );// This should be in this repo, probably
+const c_link = clone( path.resolve( __dirname, '../../components/c-link/c-link.prototype' ) );
+
+const o_nav_list_items = [
```

Comment:
I see 3 exact clones are created here, how is this being used?

Reply:
This is borderline spaghetti that's going to change once the workflow is worked out a bit more – in essence though, it is creating three copies of c-link so that they can be updated independently for demo data in another object. For example, here: https://github.com/penske-media-corp/pmc-larva/pull/11/files#diff-5c0407f092a0217cb09fd4a5a53ca3f7

Reply:
I think there could be nice helper functions for stuff like this, but it's low priority at the moment.